.github/workflows/pr-language-labeler.yml

```
@@ -0,0 +1,110 @@
+name: PR Language Labeler
+
+on:
+  pull_request:
+    types: [ opened, ready_for_review, synchronize ]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  labeling-languages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Create package.json
+        run: echo '{}' > package.json
+
+      - name: Install dependencies
+        run: npm install @octokit/rest node-fetch
+
+      - name: Detect languages and add labels
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUM: ${{ github.event.pull_request.number }}
```

nit: 요렇게 짧게 쓸 수 있는 걸로 알고 있습니다.

```suggestion
          PR_NUM: ${{ github.event.number }}
```

---

아 트리거가 pr 한정으로 되어있으니 이게 더 깔끔하겠군요! 같이 수정해두겠습니다~